hll/include/Hll4Array-internal.hpp

```diff
@@ -210,7 +210,7 @@ void Hll4Array<A>::internalHll4Update(uint32_t slotNo, uint8_t newVal) {
 
       // we just increased a pair value, so it might be time to change curMin
       if (actualOldValue == this->curMin_) { // 908
-        this->decNumAtCurMin();
+        --(this->numAtCurMin_);
```

Comment:
i'm fine with this. but having it declared as inline i'd hope the impact of the change is nothing on any modern compiler